mongodump/mongodump.go

```diff
@@ -38,22 +38,24 @@ type MongoDump struct {
 	InputOptions  *InputOptions
 	OutputOptions *OutputOptions
 
+	SkipUsersAndRoles bool
```

Comment:
This value is ignored in some cases if dump.ToolOptions.DB == "". For example: https://github.com/llvtt/mongo-tools/blob/3.3.11/mongodump/mongodump.go#L332-L337